config/resources/domain.json

```diff
@@ -521,12 +521,12 @@
           "cardinality": "many"
         },
         "can-contain-signals": {
-          "predicate": "mobiliteit:Verkeerstekenconcept.magVerkeerstekenconceptBevatten",
+          "predicate": "mobiliteit:magVerkeerstekenconceptBevatten",
```

Comment:
Following the current datamodel, this should be `mobiliteit:Verkeerstekenconcept.magVerkeerstekenconceptBevatten.target` (https://test.data.vlaanderen.be/doc/applicatieprofiel/besluit-mobiliteit/ontwerpstandaard/toolchain4/#Verkeerstekenconcept), but I might be missing something too...